runtime/compiler/z/runtime/s390_macros.inc

```diff
@@ -97,21 +97,22 @@ SETVAL(eq_cp_inInterfaceSnippet,8)
 SETVAL(eq_cpindex_inInterfaceSnippet,16)
 SETVAL(eq_intfAddr_inInterfaceSnippet,24)
 SETVAL(eq_intfMethodIndex_inInterfaceSnippet,32)
+SETVAL(eq_thunk_inInterfaceSnippet,40)
 
 ZZ For Zero or Multi slot case
-SETVAL(eq_flag_inInterfaceSnippet,40)
+SETVAL(eq_flag_inInterfaceSnippet,48)
 
 ZZ For Multi slot case
-SETVAL(eq_lastCachedSlotField_inInterfaceSnippet,48)
-SETVAL(eq_firstSlotField_inInterfaceSnippet,56)
-SETVAL(eq_lastSlotField_inInterfaceSnippet,64)
-SETVAL(eq_firstSlot_inInterfaceSnippet,72)
+SETVAL(eq_lastCachedSlotField_inInterfaceSnippet,56)
+SETVAL(eq_firstSlotField_inInterfaceSnippet,64)
+SETVAL(eq_lastSlotField_inInterfaceSnippet,72)
+SETVAL(eq_firstSlot_inInterfaceSnippet,80)
```

Comment:
Why are these defined in s390_macros.inc while all the other ones are defined in PicBuilder.m4? Can we consolidate them all into one place, i.e. PicBuilder where they are actually used?

Reply:
I moved these back to `s390_macros.inc` because a few of these are used by both the picBulder and valueProf.m4.